traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -191,6 +192,12 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		{1.3, http.MethodPost, `profiles-wip/?$`, api.CreateHandler(profile.GetRefType(), d.DB), auth.PrivLevelOperations, Authenticated, nil},
 		{1.3, http.MethodDelete, `profiles-wip/{id}$`, api.DeleteHandler(profile.GetRefType(), d.DB), auth.PrivLevelOperations, Authenticated, nil},
 
+		//ProfileParameters
+		{1.3, http.MethodGet, `profileparameters/?(\.json)?$`, api.ReadHandler(profileparameter.GetRefType(), d.DB), auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
i would recommend calling this /profile_parameters per the API guidelines - https://cwiki.apache.org/confluence/display/TC/API+Guidelines

Or maybe since it's technically working with a join table call it

/profile_parameter?

Reply:
@mitchell852 Should it be at least `/profile_parameters`?